### PR TITLE
Install instruction refactor

### PR DESCRIPTION
**Pull request by pozil**
Reviewed and updated install instructions:
- Aligned scratch org and non source-tracked orgs instructions.
- Fixes some missing instructions for DE orgs.
- Used CLI automate publishing Community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,16 +60,22 @@ E-Bikes is a sample application that demonstrates how to build applications with
     sfdx force:user:permset:assign -n ebikes
     ```
 
-1. Load sample data:
+1. Import sample data:
```
Comment (muenzpraeger) [inline README.md]: If you want this change it has to be consistent across all sample apps.

File: README.md [hunk 2]
```diff
@@ -60,16 +60,22 @@ E-Bikes is a sample application that demonstrates how to build applications with
...
 
     ```
-    sfdx force:data:tree:import --plan ./data/sample-data-plan.json
+    sfdx force:data:tree:import -p ./data/sample-data-plan.json
```
Comment (muenzpraeger) [inline README.md]: Same as above.

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ E-Bikes is a sample application that demonstrates how to build applications with
 
 -   [Installing E-Bikes using a scratch org](#installing-e-bikes-using-a-scratch-org)
 
--   [Installing E-Bikes using a Developer Edition Org](#installing-e-bikes-using-a-developer-edition-org)
+-   [Installing E-Bikes using a Developer Edition Org or a Trailhead Playground](#installing-e-bikes-using-a-developer-edition-org-or-a-trailhead-playground)
```
Comment (muenzpraeger) [inline README.md]: Good idea. Needs to be added, especially looking at the upcoming QuickStarts, to every sample app.

File: README.md [hunk 4]
```diff
@@ -60,16 +60,22 @@ E-Bikes is a sample application that demonstrates how to build applications with
...
 
     ```
-    sfdx force:mdapi:deploy -u ebikes --deploydir mdapiDeploy/unpackaged -w 5
+    sfdx force:mdapi:deploy -d mdapiDeploy/unpackaged -w 5
+    ```
+
+1. Publish the Community:
+
+    ```
+    sfdx force:community:publish -n E-Bikes
```
Comment (muenzpraeger) [inline README.md]: Awesome!

File: README.md [hunk 5]
```diff
@@ -80,38 +86,31 @@ E-Bikes is a sample application that demonstrates how to build applications with
...
     sfdx force:auth:web:login -a ebikesDE
     ```
 
-1. Check out a new branch of the code, to make changes that will allow deployment to a Developer Edition org:
-
-    ```
-    git checkout -b devOrg
-    ```
-
 1. In VS Code, use the Ctrl/Cmd-P shortcut for Quick Open. Type **E_Bikes.site** and click on the **E_Bikes.site-meta.xml** file to open it.
```
Comment (muenzpraeger) [inline README.md]: I'm wondering if we shouldn't be more clear. Cmd+P for macOS, Ctrl+P for Windows.

File: README.md [hunk 6]
```diff
@@ -122,17 +121,10 @@ These steps assume you have followed the instructions above to install the appli
     <subdomain>codeys-ebikes-developer-edition</subdomain>
     ```
 
-1. Use Quick Open again to search for **Profile** and open **E-Bikes Profile.profile**. At the very end of the file, change the value in the **\<userLicense>** line to **Guest**. Save the file.
```
Comment (muenzpraeger) [inline README.md]: Did that change with 224? That's awesome!

File: README.md [hunk 7]
```diff
@@ -177,7 +171,7 @@ This repository contains several files that are relevant if you want to integrat
 
 ### Code linting
 
-[ESLint](https://eslint.org/) is a popular JavaScript linting tool used to identify stylistic errors and erroneous constructs. To use ESLint with Visual Studio Code, install [this extension](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-lwc) from the Visual Studio Code Marketplace. The [.eslintignore](/.eslintignore) file is provided as part of this repository to exclude specific files from the linting process in the context of Lighning Web Components development.
+[ESLint](https://eslint.org/) is a popular JavaScript linting tool used to identify stylistic errors and erroneous constructs. To use ESLint with Visual Studio Code, install [this extension](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-lwc) from the Visual Studio Code Marketplace. The [.eslintignore](/.eslintignore) file is provided as part of this repository to exclude specific files from the linting process in the context of Lightning Web Components development.
```
Comment (muenzpraeger) [inline README.md]: 🤦‍♂

File: README.md (hunk 6 above)
Comment (pozil) [reply README.md]: yep, didn't have to do that.

File: README.md (hunk 3 above)
Comment (pozil) [reply README.md]: Exactly, that's the whole reason why I did this update :)

File: README.md (hunk 2 above)
Comment (pozil) [reply README.md]: Yes, we weren't consistent with commands flags in the readme. Most commands used the short flags so I decided to stick to it.

File: README.md (hunk 5 above)
Comment (pozil) [reply README.md]: Agreed, fixed.

**Pull request merged**